### ISSUE TITLE
Document why tabindex is necessary when a11y: true

### ISSUE THREAD

**Issue by julkue** [callmecavs/jump.js#60]
This would be helpful I think.

**Comment by callmecavs** [comment]
happy to look at a PR that adds this documentation to the [a11y section](https://github.com/callmecavs/jump.js#a11y) of the README.

the reason is [here]() on MDN. quoting that page:

> A negative value (usually tabindex="-1") means that the element should be focusable, but should not be reachable via sequential keyboard navigation. Mostly useful to create accessible widgets with JavaScript.

more context from @Heydon on #16

**Comment by julkue** [comment]
I do understand the meaning of tabindex, but I don't understand why a section you've just scrolled to shouldn't be reachable by keyboard navigation?

**Comment by julkue** [comment]
However, when I have time I will read through the issue you've posted.

**Comment by callmecavs** [comment]
@julmot as I understand it, `tabindex` of `-1` programmatically forces the focus.  you can still apply a `tabindex` to the element - it'll just be overridden to force focus when that element is jumped to

@Heydon does a nice job of painting the full picture in that other issue
